commands/serve/README.md

```diff
@@ -88,3 +89,49 @@ serve({
   s.close(); // close the server
 });
 ```
+
+### Micro Frontend (MFE) Mode
+
+The MFE mode, activated with the `--mfe` option, builds a visualisation bundle which is
+used in a micro frontend environemnt. The bundle is served at:
```

Comment:
```suggestion
The MFE mode, activated with the `--mfe` option, builds a visualisation which is
used in a micro frontend environemnt. The visualisation is served at:
```

Avoid possible confusion with the "Visualization bundle" that exist in Qlik Sense